propolis/src/vmm/machine.rs

```diff
@@ -188,8 +188,8 @@ pub struct MachineCtx {
 }
 
 impl MachineCtx {
```

Comment:
Doesn't have to be changed here, but is there any reason we still have `MachineCtx` which seems to just wrap an `Arc<Machine>`? `Machine` is already `pub`.

Reply:
The idea behind the distinction, at least for now, is that `MachineCtx` exposes normal "VM is running" operations, such as access to memory or the mmio/pio buses.  More uncommon operations (like directly registering new listeners on said buses) is done either during init, or through the `Chipset`, but critically, not from just any old place where you have access to a `MachineCtx` (say, though a `DispCtx`)